lib/crowbar/client/command/network.rb

```diff
@@ -0,0 +1,173 @@
+                    end
+                  end
+                end
+              end
+            end
+
+            parent.desc "Actions for real IP addresses"
+            parent.command :ip do |c|
+              c.desc "Allocate real IP address"
+              c.arg :name
+              c.arg :node
+              c.arg :network
+              c.arg :range
+              c.arg :suggestion, :optional
+              c.command :allocate do |a|
+                a.action do |global, opts, args|
+                  name = args.shift
+                  node = args.shift
+                  network = args.shift
+                  range = args.shift
+                  suggestion = args.shift || ""
+
+                  Request.instance.network_allocate_ip(name, node, network, range, suggestion) do |request|
+                    case request.code
+                    when 200
+                      say JSON.pretty_generate(request.parsed_response)
+                    else
+                      exit_now! request.parsed_response["error"]
+                    end
+                  end
+                end
+              end
+
+              c.desc "Deallocate real IP address"
+              c.arg :name
+              c.arg :node
+              c.arg :network
+              c.command :deallocate do |a|
+                a.action do |global, opts, args|
+                  name = args.shift
+                  node = args.shift
+                  network = args.shift
+
+                  Request.instance.network_deallocate_ip(name, node, network) do |request|
+                    case request.code
+                    when 200
+                      say "Successfully deallocated the IP"
+                    else
+                      exit_now! request.parsed_response["error"]
+                    end
+                  end
+                end
+              end
+            end
+
+            parent.desc "Actions for network interfaces"
+            parent.command :interface do |c|
+              c.desc "Enable a network interface"
+              c.arg :name
+              c.arg :node
+              c.arg :network
+              c.command :enable do |a|
+                a.action do |global, opts, args|
+                  name = args.shift
+                  node = args.shift
+                  network = args.shift
+
+                  Request.instance.network_enable_interface(name, node, network) do |request|
+                    case request.code
+                    when 200
+                      say JSON.pretty_generate(request.parsed_response)
+                    else
+                      exit_now! request.parsed_response["error"]
+                    end
+                  end
+                end
+              end
+
+              c.desc "Disable a network interface"
+              c.arg :name
+              c.arg :node
+              c.arg :network
+              c.command :disable do |a|
+                a.action do |global, opts, args|
```

Comment:
Unused block argument - `global`. If it's necessary, use `_` or `_global` as an argument name to indicate that it won't be used.<br>Unused block argument - `opts`. If it's necessary, use `_` or `_opts` as an argument name to indicate that it won't be used.